framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -205,6 +206,12 @@
      */
     private final Set<Class<? extends Annotation>> supportedQuals;
 
+    /**
+     * To cache the supported type qualifier names. call {@link #getSupportedTypeQualifierNames()}
+     * instead of using this field directly, as it may not have been initialized.
```

Comment:
A cross-reference to the existing `supportedQuals` would be good. Do they contain corresponding elements?